examples/manifold/plot_t_sne_perplexity.py

```diff
@@ -0,0 +1,104 @@
+"""
+=============================================================================
+ t-SNE: The effect of various perplexity values on the shape
+=============================================================================
+
+An illustration of t-SNE on the two concentric circles and the S-curve
+datasets for different perplexity values.
+
+We observe the tendency of higher variations for small perplexity values and
```

Comment:
Thank you for the review comments, @jnothman 
With "high variations" I meant that for small perplexities it is very likely that the shape may vary/change - the higher the perplexity the stable(the clear) the shape becomes across multiple runs.